File: api/redisConfig.js
// filepath: /c:/Users/James/Leaf_And_Luggage/leaf_and_luggage/api/redisConfig.js
const redis = require('redis');
const client = redis.createClient({
  socket: '/home/dojpzhqm/.redis/redis.sock' // Use the Unix socket path
});

client.on('error', (err) => {
  console.error('Redis error:', err);
});

client.on('connect', () => {
  console.log('Connected to Redis');
});

module.exports = client;

File: api/redis_config.py
# filepath: /c:/Users/James/Leaf_And_Luggage/leaf_and_luggage/api/redis_config.py
import redis

client = redis.StrictRedis(
    unix_socket_path='/home/dojpzhqm/.redis/redis.sock',
    decode_responses=True
)

try:
    client.ping()
    print("Connected to Redis")
except redis.ConnectionError as e:
    print(f"Redis error: {e}")